### PR TITLE
Fixes #402 - Show No connectivity when no Internet connection is there

### LINKED ISSUE #402: Show No connvectivity when no Internet connection is there
**Actual Behaviour**

While there is no internet connection, there is no message regarding the connectivity.

**Expected Behaviour**

Let the web app send no Connectivity message when there is not connection.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by rishiraj824**
Fixes issue #402 

Changes: 
- Adding snackbar to notify if the user is offline
- Added snackbar to notify if the user is online

Testing - 
Try to disconnect your Internet connection and connect back.

Demo Link 
http://susiinternet.surge.sh

### CODE REVIEW

**Comment by rishiraj824** [comment]
@uday96 Still working on it, will put it up soon

**Comment by rishiraj824** [comment]
@uday96 Added the Testing Link.

File: src/components/ChatApp/MessageSection/MessageSection.react.js [hunk 1]
```diff
@@ -369,6 +371,23 @@ class MessageSection extends Component {
     this._scrollToBottom();
     MessageStore.addChangeListener(this._onChange.bind(this));
     ThreadStore.addChangeListener(this._onChange.bind(this));
+    window.addEventListener('offline', this.handleOffline.bind(this));
+    window.addEventListener('online', this.handleOnline.bind(this));
+  }
+
+  handleOffline() {
+    console.log('offline')
```
Comment (isuruAb) [inline src/components/ChatApp/MessageSection/MessageSection.react.js]: remove console logs

**Comment by rishiraj824** [comment]
@isuruAb Done :+1:

**Pull request merged**